exporter/uptraceexporter/testdata/testdata.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2021 OpenTelemetry Authors
```

Comment:
Nit - We've started to remove the years from the copyright to make it easier to maintain. Could you update all files to not include it?

Reply:
Done